libvncclient/vncviewer.c

```diff
@@ -359,6 +359,10 @@ rfbClient* rfbGetClient(int bitsPerSample,int samplesPerPixel,
   client->saslSecret = NULL;
 #endif /* LIBVNCSERVER_HAVE_SASL */
 
+  client->requestedResize = FALSE;
```

Comment:
Should actually not be needed as `client` was calloc()'ed (which zeros out the memory), but it's possibly clearer like this.

Reply:
I prefer to initialize explicitly it, only for cleanliness and elegance in the code.

Reply:
Alright!